docs/index.md

```diff
@@ -1,715 +1,19 @@
-            "internal_attributes": [],
-            "lazy": [],
-            "resource": "/iot/d",
-            "service": "ul",
-            "static_attributes": [],
-            "subservice": "/mqtt"
-        }
-    ]
-}
-```
-
-#### Register a device
-
-Request
-
-```bash
-curl -i -X POST \
-  'https://iot-ready-ul.lab.fiware.org:24061/iot/devices' \
-  -H 'Content-Type: application/json' \
-  -H 'FIWARE-Service: ul' \
-  -H 'FIWARE-ServicePath: /mqtt' \
-  -d '{
-       "devices": [{
-         "device_id": "UL-MQTT-Device",
-         "protocol": "PDI-IoTA-UltraLight",
-         "transport": "MQTT",
-         "entity_name": "IoT-Ready-Entity",
-         "entity_type": "IoT-Ready-Type",
-         "attributes": [{
-         "object_id": "c",
-         "name": "count",
-         "type": "Integer" }]
-    }]
-}'
-```
-
-Response
-
-```text
-HTTP/1.1 201 Created
-Server: nginx/1.15.10
-Date: Thu, 11 Apr 2019 15:56:37 GMT
-Content-Type: application/json; charset=utf-8
-Content-Length: 2
-Connection: keep-alive
-X-Powered-By: Express
-Fiware-Correlator: 3a3bacd0-5bd1-450c-99e1-0dc33efe6f5f
-ETag: W/"2-vyGp6PvFo4RvsFtPoIWeCReyIC8"
-```
-
-#### Get a device (Optional)
-
-Request
-
-```bash
-curl -s -X GET \
-  'https://iot-ready-ul.lab.fiware.org:24061/iot/devices/UL-MQTT-Device' \
-  -H 'FIWARE-Service: ul' \
-  -H 'FIWARE-ServicePath: /mqtt' | python -m json.tool
-```
-
-Response
-
-```json
-{
-    "attributes": [
-        {
-            "name": "count",
-            "object_id": "c",
-            "type": "Integer"
-        }
-    ],
-    "commands": [],
-    "device_id": "UL-MQTT-Device",
-    "entity_name": "IoT-Ready-Entity",
-    "entity_type": "IoT-Ready-Type",
-    "lazy": [],
-    "protocol": "PDI-IoTA-UltraLight",
-    "service": "ul",
-    "service_path": "/mqtt",
-    "static_attributes": [],
-    "transport": "MQTT"
-}
-```
-
-#### Send the measurement
-
-Request
-
-```bash
-docker run -it --rm efrecon/mqtt-client pub -h iot-ready-ul.lab.fiware.org -p 27898 -m "c|1" -t "/ulmqttkey/UL-MQTT-Device/attrs"
-```
-
-#### Read the measurement
-
-Request
-
-```bash
-curl -s -X GET \
-  'https://iot-ready-ul.lab.fiware.org:21026/v2/entities?type=IoT-Ready-Type' \
-  -H 'FIWARE-Service: ul' \
-  -H 'FIWARE-ServicePath: /mqtt' | python -m json.tool
-```
-
-Response
-
-```json
-[
-    {
-        "TimeInstant": {
-            "metadata": {},
-            "type": "ISO8601",
-            "value": "2019-04-11T16:01:24.00Z"
-        },
-        "count": {
-            "metadata": {
-                "TimeInstant": {
-                    "type": "ISO8601",
-                    "value": "2019-04-11T16:01:24.484Z"
-                }
-            },
-            "type": "Integer",
-            "value": "1"
-        },
-        "id": "IoT-Ready-Entity",
-        "type": "IoT-Ready-Type"
-    }
-]
-```
+# About the program
+
+The FIWARE ecosystem means a straightforward business opportunity for IoT products: It is simple, the **FIWARE IoT
```

Comment:
It is incorrect, the FIWARE Ecosystem is the group of stakeholders, developers, universities, large corporations and so on that are using or taking into account the FIWARE technology to develop smart solutions in different vertical sectors like smart cities, smart farming, ...

Reply:
See previous comment - We use **Ecosystem** to mean the software (since we don't want to use **platform** since we are not an IoT Platform.) This complements with our consistent terminology **community** meaning the people and institutions creating that software.

So using the word  **ecosystem** is correct here.

Besides there are only so many synonyms for platform in the English Language.